src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+
+import Navbar from "../components/Navbar";
+
+function NewBeer() {
+  const [name, setName] = useState("");
+  const [tagline, setTagline] = useState("");
+  const [first_brewed, setFirst_brewed] = useState("");
+  const [attenuation_level, setAttenuation_level] = useState("");
+  const [description, setDescription] = useState("");
+  const [contributed_by, setContributed_by] = useState("");
+
+  const handleName = (e) => setName(e.target.value);
+  const handleTagline = (e) => setTagline(e.target.value);
+  const handleFirst_brewed = (e) => setFirst_brewed(e.target.value);
+  const handleAttenuation_level = (e) => setAttenuation_level(e.target.value);
+  const handleDescription = (e) => setDescription(e.target.value);
+  const handleContributed_by = (e) => setContributed_by(e.target.value);
+
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const body = {
+        name,
+        tagline,
+        first_brewed,
+        attenuation_level,
+        description,
+        contributed_by,
+      };
+
+      /* await axios.post("https://ih-beers-api2.herokuapp.com/beer", body); */
+
+      setName("");
+      setTagline("");
+      setFirst_brewed("");
+      setAttenuation_level("");
+      setDescription("");
+      setContributed_by("");
+
+      navigate("/");
+
+      axios
+        .post("https://ih-beers-api2.herokuapp.com/beers", body)
```

Comment:
Your function to create a new beer is not working because here the url you need to use is different, the correct url is:
`"https://ih-beers-api2.herokuapp.com/beers/new"`